Installation.md

```diff
@@ -3,4 +3,6 @@ All installation module should be stored here:
 
 ## Set Server to Debug to auto-update localhost
 FLASK_DEBUG=1 flask run
```

Comment:
Still in the process of setting up the installation readme so everyone can install flask and other dependencies to run the web app locally.